react/components/NumberOption.tsx

```diff
@@ -38,15 +38,22 @@ interface Props {
 
 const NumberOption: React.FC<Props> = ({ showCheckbox, ...props }) => {
   const intl = useIntl()
+  const inputNode = useRef<HTMLInputElement>(null)
```

Comment:
I prefer something like `inputRef`, but this is fine :-)

Reply:
I also prefer to stick with the convention of using the `Ref` suffix for `useRef()` objects.

Reply:
Makes sense!